### PR TITLE
add ASK payment

### PR DESCRIPTION
**Pull request by o-vartan**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

File: app/assets/images/payment_icons/ask.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 67 23" width="38" height="24" aria-labelledby="pi-ask"><title id="pi-ask">ASK</title><path d="M17.32 17.5H7.57L5.71 22H.73l9.36-21h4.8l9.39 21h-5.1l-1.86-4.5zm-1.53-3.69l-3.33-8.04-3.33 8.04h6.66zm17.929 8.55c-1.66 0-3.27-.22-4.83-.66-1.54-.46-2.78-1.05-3.72-1.77l1.65-3.66c.9.66 1.97 1.19 3.21 1.59 1.24.4 2.48.6 3.72.6 1.38 0 2.4-.2 3.06-.6.66-.42.99-.97.99-1.65 0-.5-.2-.91-.6-1.23-.38-.34-.88-.61-1.5-.81-.6-.2-1.42-.42-2.46-.66-1.6-.38-2.91-.76-3.93-1.14-1.02-.38-1.9-.99-2.64-1.83-.72-.84-1.08-1.96-1.08-3.36 0-1.22.33-2.32.99-3.3.66-1 1.65-1.79 2.97-2.37 1.34-.58 2.97-.87 4.89-.87 1.34 0 2.65.16 3.93.48 1.28.32 2.4.78 3.36 1.38l-1.5 3.69c-1.94-1.1-3.88-1.65-5.82-1.65-1.36 0-2.37.22-3.03.66-.64.44-.96 1.02-.96 1.74s.37 1.26 1.11 1.62c.76.34 1.91.68 3.45 1.02 1.6.38 2.91.76 3.93 1.14 1.02.38 1.89.98 2.61 1.8.74.82 1.11 1.93 1.11 3.33 0 1.2-.34 2.3-1.02 3.3-.66.98-1.66 1.76-3 2.34-1.34.58-2.97.87-4.89.87zm20.24-8.61l-2.82 2.94V22h-4.83V1h4.83v9.81L60.44 1h5.4l-8.7 9.36L66.35 22h-5.67l-6.72-8.25z" fill="#05AFF2"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/ask.svg:1]: Hi ,

Please follow the [contributions](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) guideline:
- The background is missing
- The border of the icon is missing

You can follow this [example icon](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg) as a reference.

**Review by tauthomas01 — CHANGES_REQUESTED** [review]

File: app/assets/images/payment_icons/ask.svg (hunk 1 above)
Comment (o-vartan) [reply app/assets/images/payment_icons/ask.svg]: changes applied

Comment (o-vartan) [reply app/assets/images/payment_icons/ask.svg]: changes applied

**Comment by o-vartan** [comment]
Could you assist with our pool request and indicate status of processing?
If you have any questions, please contact Hunter Jensen <hjensen@permission.io>

File: app/assets/images/payment_icons/ask.svg [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            id="b2iXXQyQoY"
+        ></path>
+    </defs>
+    <g>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/ask.svg:11]: - Can we remove the unused `<g>`?
- If it's not needed, can we also remove the `<defs>` above?

Once the changes are done, you will need to run the code through `svgo`. You can read more about it in the [contributions](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) guideline.

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Hi,

I left some comments.

Please let me know if there are any questions.

**Comment by o-vartan** [comment]
changes are made and run through svgo

**Comment by barefootdev** [comment]
Does anything else need to be done to get this pull request merged?

**Comment by PolinaSalimyanova** [comment]
@tauthomas01  Could you assist to resolve the conflicting files? What should we do from our side?

**Comment by tauthomas01** [comment]
> @tauthomas01 Could you assist to resolve the conflicting files? What should we do from our side?

@PolinaSalimyanova You will need to rebase your branch off `master`.
Also, the SVG code needs to be run through svgo, including removing whitespace. 

See this example: 
https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg?short_path=e20b466

**Comment by PolinaSalimyanova** [comment]
Hi
Could you assist with our merges?
It seems that we solved the issues but the result is failed

Thank you

Polina Salimyanova
Project Manager
SolbegSoft
[cid:image002.png@01D6A71B.2EF48260]
Email: polina.salimyanova@solbeg.com<mailto:polina.salimyanova@solbeg.com>
Phone: +35293690056

**Comment by tauthomas01** [comment]
The icon looks good. Please rebase the branch off `master` and fix the merge conflict.

**Comment by larouxn** [comment]
I think we may be able to close this PR as it seems to be a duplicate of https://github.com/activemerchant/payment_icons/pull/371.

**Comment by larouxn** [comment]
Closing as this icon has already been added via https://github.com/activemerchant/payment_icons/pull/371.

**Pull request closed without merging**